daemon/snap.go

```diff
@@ -41,9 +42,13 @@ type aboutSnap struct {
 	info   *snap.Info
 	snapst *snapstate.SnapState
 	health *client.SnapHealth
+
+	UserHold   time.Time `json:"user-hold,omitempty"`
+	GatingHold time.Time `json:"gating-hold,omitempty"`
```

Comment:
same comments about naming, also why are these exported? the other fields in this structure are not